packages/frontend/src/pages/Template/components/TemplateStepContent.tsx

```diff
@@ -58,23 +66,20 @@ export default function TemplateStepContent(props: TemplateStepContentProps) {
               color="base.content.default"
             />
```

Comment:
```suggestion
              ml={2}
```
can add this to the fallback icon since main icon has this, it ensures that the margin is the same to reduce the shifting of the icons

Reply:
thks for the catch